pathml/core/slide_data.py

```diff
@@ -336,7 +338,12 @@ def shape(self):
         Returns:
             Tuple[int, int]: Shape of image (H, W)
         """
-        return self.slide.get_image_shape()
+        if self.backend == "h5path":
```

Comment:
We could add the image shape to our .h5path specification?

Reply:
added